src/learn/learn.cpp

```diff
@@ -1987,9 +1987,6 @@ namespace Learner
         Eval::NNUE::InitializeTraining(eta1, eta1_epoch, eta2, eta2_epoch, eta3);
         Eval::NNUE::SetBatchSize(nn_batch_size);
         Eval::NNUE::SetOptions(nn_options);
-        if (newbob_decay != 1.0 && !Options["SkipLoadingEval"]) {
-            learn_think.best_nn_directory = std::string(Options["EvalDir"]);
-        }
```

Comment:
We need to set the folder path containing a original net file to `best_nn_direcoty`.  Because the trainer will load the original net file from the `best_nn_direcoty` folder, restore the network parameters and resume training if the loss is not decreased.  Maybe we could remove these lines in this pull request, fix them in another pull request.

Reply:
Ah, I see, the trainer doesn't use the mechanism of the official-stockfish/player to find the net (i.e. look first the the current working directory for the net, afterwards near the binary, etc.). I don't know the learner/trainer code well enough to fix the logic... 

I wonder if the regression we have is related to which net is loaded at which point. This has changed quite a bit in master (see `init_NNUE()`) and have some logic to protect against mistakes (see `verify_NNUE()`)

Reply:
I'm also investigating the regression.  It seems that there are two regressions.  But I will share the information after I prepare detail materials.